developing/direnv/README.md

```diff
@@ -8,7 +8,9 @@
 
 Via homebrew:
 
-     brew install direnv
+```
+ brew install direnv
```

Comment:
There is an extra space here before `brew`.

Reply:
That is curious. If I remove it manually, mdformat doesn't restore it. I'll follow up.

Reply:
OK, got a quick dev [response](https://github.com/executablebooks/mdformat/issues/272).

It looks like there was an extra space to begin with. Have a look at [the file](https://github.com/trussworks/Engineering-Playbook/tree/882baa776df02b2f39fa09b1181570bde069def7/developing/direnv) before mdformat got to it and the extra space is there too. It's just more visible when in a fenced code block.